folly/ssl/OpenSSLVersionFinder.h

```diff
@@ -18,14 +18,20 @@
 #include <folly/Conv.h>
 #include <folly/portability/OpenSSL.h>
 
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#define OPENSSL_VERSION SSLEAY_VERSION
```

Comment:
These symbols should be namespaced to folly.

Reply:
Not sure what you mean (I don't really know C++).

Reply:
`#define FOLLY_DETAIL_OPENSSL_VERSION ...`.

And then `#undef` them after using them (preferred) or at the bottom of the file.

Reply:
Prefer to have this in the OpenSSL portability header so that everything is uniform.